tests/cross_fw/examples/run_example.py

```diff
@@ -176,6 +176,14 @@ def llm_tune_params() -> Dict[str, float]:
     return {"awq": bool(awq), "ratio": ratio, "group_size": group_size}
 
 
+def llm_compression_synthetic() -> Dict[str, float]:
+    from examples.llm_compression.openvino.tiny_llama_synthetic_data.main import main as llm_compression_synthetic_main
+
+    result = llm_compression_synthetic_main()
+
+    return {"wikitext_based_similarity": result[0], "synthetic_based_similarity": result[1]}
+
+
```

Comment:
This example run would increase the overall time for the test_examples scope significantly. Only this example would take ~30 minutes to run on the local machine.
@alexsu52, are you sure that we should add this into the scope?

Reply:
More precise example timings, @alexsu52, @MaximProshin 
![image](https://github.com/user-attachments/assets/f5d363f0-8cb7-4198-8589-41a4a5951318)
~30 minutes take the example in the test environment including package installation & etc.

Reply:
The reduction of the subset & dataset size to 64, reduces the time from ~20 mins to ~16 mins for the example
![image](https://github.com/user-attachments/assets/65caf360-1d86-40ad-9a11-68953ccf3149)